src/api/objects/ObjectAPI.js

```diff
@@ -168,6 +168,61 @@ ObjectAPI.prototype.get = function (identifier) {
     });
 };
 
+ObjectAPI.prototype.deSynchronize = function (identifier) {
```

Comment:
I've proposed some changes in the general comments that I think will reduce the amount of new API necessary here, so we may not need these functions with the revised approach.

Reply:
Removed